### DISCUSSION TITLE
Tag source code as packaged version

### DISCUSSION THREAD

**Discussion by Gounlaf** [dlemstra/Magick.Native#44] [Ideas]
:wave:

Is it possible to tag sources with same version as packages? 
I don't know what's the best way to do it because I see that npm & nuget package versions doesn't follow the same pattern.

I'm not familiar with github workflow so I can't make a PR for that =/

Thanks.

Regards.

**Comment by dlemstra** [comment]
I don't want to add tags because I don't need them. The build now creates a new package for every build on GitHub actions. And that would mean a lot of tags would be added.

**Reply by Gounlaf** [reply]
Can I assume that latest package always correspond to latest commit? (except while the pipeline is running)

**Reply by dlemstra** [reply]
You can assume that.
